samples/redirect.cc

```diff
@@ -32,7 +32,7 @@ void rot13_io(void)
 {
     std::string s;
 
-    std::cin >> s;
+    std::getline(std::cin, s);
```

Comment:
That's a great catch! Since we didn't inspect the rest of the string before, this went unnoticed.